packages/nimble/inst/include/nimble/accessorClasses.h

```diff
@@ -161,7 +161,8 @@ class SingleVariableMapAccessBase {
  public:
   int offset, length;
   bool singleton;
-  vector<int> sizes, strides; 
+  vector<int> sizes, strides;
+ SingleVariableMapAccessBase() : offset(0), singleton(false) {length = 0;}; /* length(0) triggers the Rf_length macro mixup */
```

Comment:
length(0) triggers error from R's macro `length` which replaces `length(x)` with `Rf_length(x)`.  You're getting us fixed on that but meanwhile `length(0)` breaks.